src/Components/AuthInput/index.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './styles.module.scss';
+import { handleAuthInputStyles } from '../../utils';
+
+
+export const AuthInput = React.memo(function AuthInput({
```

Comment:
is it any reaon to have this component an `AuthInput`. you should try keep it as much abstract as possible so please consider introducing some `Input` component that can be re-used for any input. Any auth-related logic should not be put into UI component